json_text.js

```diff
@@ -490,7 +522,9 @@ arrPopups = [
             "Bern Kantonu: Hava kirliliği kontrolüne ilişkin ipuçları (" +
             "<a href=\"https://www.vol.be.ch/vol/de/index/umwelt/luftreinhaltung/tipps_zur_luftreinhaltung.html\" target=\"_blank\">Almanca</a>/" +
             "<a href=\"https://www.vol.be.ch/vol/fr/index/umwelt/luftreinhaltung/tipps_zur_luftreinhaltung.html\" target=\"_blank\">Fransızca</a>/" +

```

Comment:
Dieser Link stimmt nicht. Es geht nicht die französische Seite auf. Das müsste man aber auch in den anderen Sprachen ändern.

Reply:
Stimmt. wird in separatem request für alle Sprachen angeschaut.